scripts/distances/align.py

```diff
@@ -0,0 +1,663 @@
+from collections import Counter
+import math
+import distance
+from munkres import Munkres, print_matrix
+import numpy as np
+from textutil import break2sentences, extract_terminals
+import evaluation
+import re
+import string
+import zss
+from ucca import layer0, layer1
+
+import inspect
+
+def compare(n1, n2):
+	return True if labels(n1) & labels(n2) else False
```

Comment:
this is like  bool(labels(n1) & labels(n2)


On Sat, Jun 18, 2016 at 8:54 PM, Daniel Hershcovich <
notifications@github.com> wrote:

> In scripts/distances/align.py
> <https://github.com/danielhers/ucca/pull/26#discussion_r67604285>:
>
> > +from collections import Counter
> > +import math
> > +import distance
> > +from munkres import Munkres, print_matrix
> > +import numpy as np
> > +from textutil import break2sentences, extract_terminals
> > +import evaluation
> > +import re
> > +import string
> > +import zss
> > +from ucca import layer0, layer1
> > +
> > +import inspect
> > +
> > +def compare(n1, n2):
> > +	return True if labels(n1) & labels(n2) else False
>
> This is just like returning labels(n1) & labels(n2), isn't it? Or maybe bool(labels(n1)
> & labels(n2)).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danielhers/ucca/pull/26/files/791cc894129d9f155144e79fa649a75a65f2c340#r67604285>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGHo2CMRqHWXIi8_NLavdlpckBCD8J_jks5qNDDBgaJpZM4I46t3>
> .
>